### DISCUSSION TITLE
Difference user field layout (UserTypes)

### DISCUSSION THREAD

**Discussion by alexanderloewe** [craftcms/cms#13026] [Ideas] [answered]
I often have the situation that I need/want to enter different types of users as users in the Craft CMS. Partly, the users have different needs for fields. 

Which solution is technically the best is hard to say. What I would suggest as an approach would be to solve it as with the Sctions via EntryTypes, in this case quasi UserTypes, so you could better represent and manage the data that belongs to a user. In the representation in the backend, but of course also in the front-end, because you could control different forms and inputs via the type.

**Comment by brandonkelly** [comment, marked as answer]
You can accomplish this with user groups. In your user field layout, you can add conditions to tabs/fields that prevent the tab/field from showing unless the user belongs to a certain user group(s).
